File: Unidad_III/Brandon/ejemplo_2/Java/src/Agencia.java
import java.util.Scanner;


public class Agencia {

	public static void main(String[] args) {
		
		String marcaM,modeloM,colorM,temp;
		int yearM,kiloM;
		
		Auto auto1= new Auto();
		
		Scanner leer= new Scanner(System.in);
		
		System.out.println("Ingresa la marca");
		marcaM=leer.nextLine();
		System.out.println("ingresa el modelo");
		modeloM=leer.nextLine();
		System.out.println("Ingresa el año");
		yearM=leer.nextInt();
		System.out.println("Ingresa el Kilometraje");
		kiloM=leer.nextInt();
		System.out.println("Ingrese el color");
		colorM=leer.nextLine();
		
		
		Auto auto2= new Auto(marcaM,modeloM,yearM,kiloM,"Azul");
		
		auto1.Checar();
		auto2.Checar();
		///////////////////////////////////////////////////////
		System.out.println("Ingrese el color nuevo");
		colorM=leer.nextLine();
		
		auto2.Pintar(colorM);
		////////////////////////////////////////////////////////
		System.out.println("Ingresa el Kilometraje nuevo");
		kiloM=leer.nextInt();
		
		auto2.Correr(kiloM);
		///////////////////////////////////////////////////////
		System.out.println("ingresa el modelo nuevo");
		modeloM=leer.nextLine();
		
		auto2.Modificar(modeloM);
		///////////////////////////////////////////////////////
		auto2.Checar();
		
		

	}

}

File: Unidad_III/Felipe/C++/src/Agencia.cpp
#include <Auto.hpp>


using namespace std;

int main()
{
    string Cmarca, Cmodelo, Ccolor;
    int Cyear, Ckilometros;
    
    cout << "Ingrese la marca" << std::endl;
    cin >> Cmarca;
    cout << "Ingrese el modelo" << std::endl;
    cin >> Cmodelo;
    cout << "Ingrese el año" << std::endl;
    cin >> Cyear;
    cout << "Ingrese el kilometraje" << std::endl;
    cin >> Ckilometros;
    cout << "Ingrese el color" << std::endl;
    cin >> Ccolor;
    
    Auto auto1;
    
     Auto* auto2 = new Auto(Cmarca,Cmodelo,Cyear,Ckilometros,Ccolor);
    
     auto1.checar();
     auto2->checar();

    cout << "Ingrese el nuevo color" << endl;
    cin >> Ccolor;
    auto2->pintar(Ccolor);

    cout << "Ingrese cuanto corrió" << endl;
    cin >> Ckilometros;
    auto2->correr(Ckilometros);

    cout << "Ingrese el nuevo modelo" << endl;
    cin >> Cmodelo;
    auto2->modificar(Cmodelo);

    auto2->checar();

return 0;

    
}